bugsnag-android-core/src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -34,7 +34,8 @@ private Bugsnag() {
      */
     @NonNull
     public static Client init(@NonNull Context androidContext) {
-        return init(androidContext, null, true);
+        String apiKey = null;
```

Comment:
what's the purpose of this change?

Reply:
The change should be functionally equivalent - `autoNotify` is true by default so can be removed. We need to specify that `apiKey` is a `String` explicitly, as `init` has two methods that would otherwise be ambiguous if we passed null:

```
init(context, string)
init(context, configuration)
```